client/src/styles/legacy/_style.scss

```diff
@@ -791,44 +713,18 @@ html, body {
   overflow: hidden;
   z-index: 100000;
   background: $body-bg;
+  opacity: 0;
+  transition: opacity .5s ease-in-out;
 
-  &__logo {
-    &__part {
-      fill: $blue;
-      stroke: $blue;
-      stroke-width: 1;
-      stroke-linejoin: miter;
-      stroke-dasharray: 225;
-      stroke-dashoffset: 0;
-
-      animation: logo__draw 2s cubic-bezier(.445, .05, .55, .95) 2 alternate both 1s;
-
-      @keyframes logo__draw {
-        0% {
-          fill-opacity: 1;
-          stroke-dashoffset: 0;
-        }
-        40% {
-          fill-opacity: 0;
-          stroke-dashoffset: 0;
-        }
-        90% {
-          fill-opacity: 0;
-          stroke-dashoffset: 225;
-        }
-        100% {
-          fill-opacity: 0;
-          stroke-dashoffset: 225;
-        }
-      }
-    }
+  body.loading & {
+    opacity: 1;
```

Comment:
This snippet adds an opacity transition when the `loading` class is removed from the `<body>`.